src/plugins/GUITestBase/src/tests/PosteriorActions.cpp

```diff
@@ -89,17 +75,27 @@ POSTERIOR_ACTION_DEFINITION(post_action_0001) {
     // Close all popup widgets
     // Close all modal widgets
     // Clear the clipboard
-#ifdef Q_OS_MAC
-    workaroundForMacCGEvents();
-#endif
+
     QWidget *popupWidget = QApplication::activePopupWidget();
     while (popupWidget != NULL) {
+
+#ifdef Q_OS_MAC
+    GTUtilsMac fakeClock;
+    fakeClock.startWorkaroundForMacCGEvents(1, true);
+    fakeClock.startWorkaroundForMacCGEvents(16000, false);
+#endif
         GTWidget::close(os, popupWidget);
         popupWidget = QApplication::activePopupWidget();
     }
 
     QWidget *modalWidget = QApplication::activeModalWidget();
     while (modalWidget != NULL) {
+
+#ifdef Q_OS_MAC
```

Comment:
nit: code ident looks inconsistent for Q_OS_MAC blocks

Reply:
fixed